docs/relational-databases/performance/tune-performance-with-the-query-store.md

```diff
@@ -105,7 +112,25 @@ ORDER BY rs.avg_physical_io_reads DESC;
 
 #### Queries with multiple plans
 
-These queries are especially interesting because they're candidates for regressions due to plan choice change. The following query identifies these queries along with all plans:
+These queries are especially interesting because they're candidates for regressions due to plan choice change. 
```

Comment:
Suggestion: "these queries" is vague...  What about "Queries with more than one plan are especially interesting because they can be candidates for a regression in performance due to a change in plan choice."